modules/ROOT/pages/to-configure-api-settings.adoc

```diff
@@ -4,28 +4,18 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-Anypoint Exchange lets you configure whether REST, SOAP, and HTTP API services are public or private in a Private Exchange or in an Exchange Portal. Typical services that you would set visibility for are proxies or policies associated with your API that you set using API Manager.
+Use Anypoint Exchange to configure whether API instances are public or private in a private Exchange or in an Exchange portal. For example, configure public or private visibility for proxies or policies associated with your API and set using API Manager.
```

Comment:
```suggestion
Use Anypoint Exchange to configure whether API instances are public or private in a private Exchange server or in an Exchange portal. For example, configure public or private visibility for proxies or policies associated with your API and set using API Manager.
```
GLOBALLY avoid using trademarks as nouns preceded by articles (a, the); provide a common noun object after them, making them adjectives, per style ref. and trademarking rules.

Reply:
Good point! Fixed.